### PR TITLE
Simplify hardswish ONNX export graph.

### LINKED ISSUE #58301: HardSwish ONNX export is not optimized.
## 🐛 Bug

HardSwish ONNX export has been added in #48423, but it is exported as a series of primitive ops (Add, Clip, Div). ONNX already has the HardSigmoid op since opset 6 and will have a dedicated HardSwish op since opset 14. For now, we can change it to export as `x * HardSigmoid(x)` for simplicity.

In fact, nn.HardSigmoid() op has already been exported as HardSigmoid op directly in #54193. Please use this as a reference for fixing HardSwish.

On the practical side, when exporting MobileNetV3 to ONNX, currently all the HardSwish activations are exported as primitives, which makes the graph very big.

Screen Shot 2021-05-14 at 9 25 49 AM

## Expected behavior

Change to export nn.HardSwish() as `x * HardSigmoid(x)` directly instead of primitive ops to reduce ONNX graph size.

Screen Shot 2021-05-14 at 9 26 40 AM

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof @SplitInfinity

### PR DESCRIPTION
**Pull request by PeterL1n**
Fixes #58301

### CODE REVIEW

**Review by shubhambhokare1 — CHANGES_REQUESTED** [review]
Overall looks good, please also remove the _hardtanh_helper function present in symbolic_helper.py

**Comment by PeterL1n** [comment]
@shubhambhokare1 Good catch. Done.

**Review by shubhambhokare1 — APPROVED** [review]
LGTM! We need to wait for CI testing though.
@BowenBao do we need another approval for the CI testing?

**Review by garymm — CHANGES_REQUESTED** [review]

File: torch/onnx/symbolic_opset9.py
Comment (PeterL1n) [reply torch/onnx/symbolic_opset9.py]: Changed to use double quote.

I don't think a comment is necessary. 1/6 is always PyTorch's hardsigmoid's definition. You can refer to the hardsigmoid export function right below. No comment was included.

File: torch/onnx/symbolic_opset9.py
Comment (garymm) [inline torch/onnx/symbolic_opset9.py:1918]: I realize you didn't originally write this code, but since you're touching related code, let's try to improve it.
Please add the comment to hardsigmoid below.

The comment can be something like:
```
# 1 / 6 makes op equivalent to PyTorch.
# See https://pytorch.org/docs/stable/generated/torch.nn.Hardsigmoid.html
```

File: torch/onnx/symbolic_opset9.py
Comment (garymm) [inline torch/onnx/symbolic_opset9.py:1914]: Let's re-use the existing hardsigmoid function rather than duplicate it here. That avoids duplication of the alpha_f constant and the ONNX op name.

```
hs = hardsigmoid(g, self)
return g.op("Mul", self, hs)
```

**Review by garymm — CHANGES_REQUESTED** [review]

File: torch/onnx/symbolic_opset9.py
Comment (PeterL1n) [reply torch/onnx/symbolic_opset9.py:1918]: Done!

File: torch/onnx/symbolic_opset9.py
Comment (PeterL1n) [reply torch/onnx/symbolic_opset9.py]: Done!

**Review by garymm — APPROVED** [review]

**Comment by garymm** [comment]
@SplitInfinity could you please import this?

**Comment by SplitInfinity** [comment]
@garymm 

I tried to import this but couldn't get our internal tests to run properly. I *think* it might have something to do with the merge conflicts I now see in the GitHub UI. Maybe they showed up while you were waiting for me to import?

**Comment by PeterL1n** [comment]
I did a merge instead of a rebase. @SplitInfinity

**Pull request closed without merging**
